jiup/rules/rules.go

```diff
@@ -457,6 +457,18 @@ func init() {
 			return &x86, &x64, nil
 		},
 	)
+	Rule("empoche",
+		v.HTML(
+			"https://empoche.com/download",
+			"#tab-windows",
+			"innerText",
+			h.Re("Empoche\\+Setup\\+([0-9.]+).exe"),
+		),
+		d.Template(
+			"https://empoche-desktop.s3.eu-central-1.amazonaws.com/Empoche+Setup+{{.Version}}.exe",
+			"",
+		),
```

Comment:
I'd rather use the HTMLA download extractor here.

Reply:
Is it possible to write a nice match for this one?

Reply:
`#tab-windows a[href$=".exe"]`. Remember that it uses [Cascadia](https://github.com/andybalholm/cascadia) for matches.